dotnet/src/dotnetframework/GxClasses/Helpers/HttpHelper.cs

```diff
@@ -78,6 +78,9 @@ public static void SetResponseStatus(HttpContext httpContext, string statusCode,
 					case "201":
 						wcfcontext.OutgoingResponse.StatusCode = HttpStatusCode.Created;
 						break;
+					case "202":
```

Comment:
I see you have two almost identical switch blocks, the difference is where the resulting value is assigned.

Consider creating a method that receives the string status code and returns a `HttpStatusCode`. Then you can assign it to `wcfcontext.OutgoingResponse.StatusCode` or cast it to `int` and assign it to `httpContext.Response.StatusCode`.

Reply:
Done.